pkg/madmin/info-commands.go

```diff
@@ -147,3 +150,51 @@ func (adm *AdminClient) ServerInfo() ([]ServerInfo, error) {
 
 	return serversInfo, nil
 }
+
+// ServerDrivesPerfInfo holds informantion about address and write speed of
+// all drives in a single server node
+type ServerDrivesPerfInfo struct {
+	Addr  string             `json:"addr"`
+	Error string             `json:"error,omitempty"`
+	Perf  []disk.Performance `json:"perf"`
+}
+
+// PerfType represents the type of performance
+// information to be fetched.
+// Supported values = "drive"
+type PerfType string
+
+// ServerDrivesPerfInfo - server's endpoints performance info based on input PerfType
+func (adm *AdminClient) ServerDrivesPerfInfo(perfType PerfType) ([]ServerDrivesPerfInfo, error) {
```

Comment:
You don't need to take drive as perf type because this function is called ServerDrivesPerfInfo

Reply:
done